tests/data/test-data.sql

```diff
@@ -5,7 +5,10 @@ insert into login (id, login, email, password, create_time) values
     (100, 'ladder1', 'ladder1@example.com', SHA2('ladder1', 256), '2000-01-01 00:00:00'),
     (101, 'ladder2', 'ladder2@example.com', SHA2('ladder2', 256), '2000-01-01 00:00:00'),
     (102, 'ladder_ban', 'ladder_ban@example.com', SHA2('ladder_ban', 256), '2000-01-01 00:00:00'),
-    (200, 'banme', 'banme@example.com', SHA2('banme', 256), '2000-01-01 00:00:00')
+    (200, 'banme', 'banme@example.com', SHA2('banme', 256), '2000-01-01 00:00:00'),
+    (201, 'ban_revoked', 'ban_revoked@example.com', SHA2('ban_revoked', 256), '2000-01-01 00:00:00'),
+    (202, 'ban_expired', 'ban_expired@example.com', SHA2('ban_expired', 256), '2000-01-01 00:00:00'),
+    (203, 'ban_long_time', 'ban_null_expiration@example.com', SHA2('ban_long_time', 256), '2000-01-01 00:00:00')
```

Comment:
you can leave `,` at the end of row :-)

Reply:
In Python yes, but in SQL, sadly no you can't. Believe me I already tried. SQL is very picky about its syntax